packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js

```diff
@@ -79,66 +83,68 @@ class Checkout extends Page {
      * @param {String} checkout.isValid The checkout validation
      */
 
-    open(checkout){
-        let authQueryString = ''; 
+    open (checkout) {
+        let authQueryString = '';
         let createGuestUrl = '&knob-Create%20Guest%20Url=%2Fcreate-guest.json';
-        let checkoutUrl = checkout.isValid ? `&knob-Get%20Checkout%20Url=%2Fcheckout-${checkout.type}.json` : '';
-        let basketUrl = `&knob-Get%20Basket%20Url=%2Fget-basket-${checkout.type}.json`
+        const checkoutUrl = checkout.isValid ? `&knob-Get%20Checkout%20Url=%2Fcheckout-${checkout.type}.json` : '';
+        const basketUrl = `&knob-Get%20Basket%20Url=%2Fget-basket-${checkout.type}.json`;
+        const placeOrderUrl = '&knob-Place%20Order%20Url=%2Fplace-order.json';
+        const paymentPageUrlPrefix = '&knob-Payment%20Page%20Url%20Prefix=%23%2Fpay';
 
-        if ( checkout.isAuthenticated ) {
-            authQueryString = '&knob-Auth%20token=a'; 
+        if (checkout.isAuthenticated) {
+            authQueryString = '&knob-Auth%20token=a';
             createGuestUrl = '';
         }
-        
-        let url = `checkout-component${checkoutUrl}&knob-Available%20Fulfilment%20Url=%2Fcheckout-available-fulfilment.json${authQueryString}${basketUrl}${createGuestUrl}`
+
+        const url = `checkout-component${checkoutUrl}&knob-Available%20Fulfilment%20Url=%2Fcheckout-available-fulfilment.json${authQueryString}${basketUrl}${createGuestUrl}${placeOrderUrl}${paymentPageUrlPrefix}`;
```

Comment:
Lines 91, 92 and 99 are the only ones I've changed. The other changes are linting fixes.